Azure/sonic-buildimage
# Frr 8.2 upgrade

#### Why I did it
Upgrade FRR to version 8.2.2. Build libyang2 required by FRR.

#### How I did it
Update FRR version and tag.

#### How to verify it
Following tests were performed on sonic-vs:
- BGP docker status check
- BGP configuration and session establishment
- Route redistribution and ping
- Issued show commands to check the bgp neighbor and routes
- Checked app-db to ensure bgp routes are installed with correct interface and nexthop.
- Create VRF and check FRR knows the VRF
- Check VRF routes are installed in app-db with correct Vrf name and next-hop
- Establish BGP Evpn session and check if Evpn routes (multicast, mac, prefix) are exchanged and installed correctly in app-db.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
